README.md

```diff
@@ -1,55 +1,64 @@
-# browser-compat-data
+# mdn-browser-compat-data
 
 [https://github.com/mdn/browser-compat-data](https://github.com/mdn/browser-compat-data)
 
-Maintained by the [MDN team at Mozilla](https://wiki.mozilla.org/MDN).
-
 This repository contains compatibility data for Web technologies.
 Browser compatibility data describes which platforms (where "platforms" are
 usually, but not always, web browsers) support particular Web APIs.
 
 This data can be used in documentation, to build compatibility tables listing
 browser support for APIs. For example:
-[Browser support for JavaScript APIs](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Browser_support_for_JavaScript_APIs).
+[Browser support for WebExtension APIs](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Browser_support_for_JavaScript_APIs).
+
+[![npm](https://img.shields.io/npm/v/mdn-browser-compat-data.svg)](https://www.npmjs.com/package/mdn-browser-compat-data)
+[![Build Status](https://travis-ci.org/mdn/browser-compat-data.svg?branch=master)](https://travis-ci.org/mdn/browser-compat-data)
+[![Twitter Follow](https://img.shields.io/twitter/follow/mozdevnet.svg?style=social&label=Follow&style=plastic)](https://twitter.com/MozDevNet)
+
+Maintained by the [MDN team at Mozilla](https://wiki.mozilla.org/MDN).
+
+## Installation
+You can install mdn-browser-compat-data as a node package ([tutorial](https://www.npmjs.com/package/mdn-browser-compat-data/tutorial)).
+```
+npm install mdn-browser-compat-data
+```
+
+## Usage
+```js
+const bcd = require('mdn-browser-compat-data');
+bcd.data.css.properties.background;
+// returns a compat data object (see schema)
+```
```

Comment:
This readme is also the readme appearing here: https://www.npmjs.com/package/mdn-browser-compat-data. I saw this quite often on other GitHub repositories/npm packages, too.